dao/src/main/java/org/thingsboard/server/dao/timeseries/CassandraBaseTimeseriesDao.java

```diff
@@ -591,6 +561,56 @@ private PreparedStatement getDeleteStmt() {
         return deleteStmt;
     }
 
+    private PreparedStatement getSaveWithNullStmt() {
+        if (saveWithNullStmt == null) {
+            stmtCreationLock.lock();
+            try {
+                if (saveWithNullStmt == null) {
+                    saveWithNullStmt = prepare(INSERT_INTO + ModelConstants.TS_KV_CF +
```

Comment:
Please extract this to a separate constant (C) and use it in the next method (C+" USING TTL ?")